### ISSUE TITLE
Doctests fail to run because of relative imports

### ISSUE THREAD

**Issue by JLBegin** [DCC-Lab/RayTracing#214]
Error example : 
```
    from .matrixgroup import *
ImportError: attempted relative import with no known parent package
```
Here's a [related SO post](https://stackoverflow.com/questions/4047227/doctest-and-relative-imports) where the best solution seems to be to replace relative imports with absolute imports.

**Comment by dccote** [comment]
> Here's a related SO post where the best solution seems to be to replace relative imports with absolute imports.

We can't really do that, hat will break the module.

**Comment by dccote** [comment]
So, should we close this or we will solve it?

**Comment by dccote** [comment]
Fixed by Elahe #305
